ports/stm32/usbd_conf.c

```diff
@@ -127,10 +128,16 @@ void HAL_PCD_MspInit(PCD_HandleTypeDef *hpcd) {
         #endif
     }
     #if MICROPY_HW_USB_HS
-    else if (hpcd->Instance == USB_OTG_HS) {
+    else
+    #endif
+    #endif
+    #if MICROPY_HW_USB_HS
```

Comment:
instead of this guarded `else`, would be simpler to just put a `return;` at the end of the `USB_OTG_FS` if block